src/modules/dipendenti-in-cloud/dipendenti-in-cloud-utils.ts

```
@@ -0,0 +1,49 @@
+      }
+      // keep reading chunks...
+      if (item?.text) {
+        items.push(item);
+      }
+      // until they finish
+      if (!item) {
+        // we read the whole pdf
+        resolve(items as DataEntry & { x: number; y: number }[]); // items type is wrong
+      }
+    });
+  });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)